src/libs/AxelarQueryAPI.ts

```diff
@@ -22,6 +22,22 @@ interface TranslatedTransferRateLimitResponse {
   outgoing: string;
   limit: string;
 }
+interface GMPExpressTransactionParameters {
+  isGMPExpressTransaction: boolean;
+  transferAmount: BigNumber;
```

Comment:
axelarscan's API interprets the amount in terms of the symbol instead of unit denom, so this could be a big float too, not sure if this type covers that